Modules/Core/Common/include/itkBSplineInterpolationWeightFunction.h

```diff
@@ -112,15 +113,19 @@ class ITK_TEMPLATE_EXPORT BSplineInterpolationWeightFunction
 #endif
 
 protected:
-  BSplineInterpolationWeightFunction();
+  BSplineInterpolationWeightFunction() = default;
   ~BSplineInterpolationWeightFunction() override = default;
 
 private:
   /** Lookup table type. */
-  using TableType = Array2D<unsigned int>;
+  using TableType = FixedArray<IndexType, NumberOfWeights>;
 
   /** Table mapping linear offset to indices. */
-  TableType m_OffsetToIndexTable;
+  const TableType m_OffsetToIndexTable{ [] {
+    TableType     table;
+    std::copy_n(ZeroBasedIndexRange<SpaceDimension>(SupportSize).cbegin(), NumberOfWeights, table.begin());
```

Comment:
Modules/Core/Common/include/itkBSplineInterpolationWeightFunction.h:125:17: error: 'ZeroBasedIndexRange' was not declared in this scope

When an address of a `static constexpr symbol` is taken, it needs to be defined somewhere in the translation unit, hence: `itkBSplineInterpolationWeightFunctionTest.cxx:(.text+0x2539): undefined reference to `itk::BSplineInterpolationWeightFunction<double, 3u, 3u>::SupportSize'`

Reply:
> Modules/Core/Common/include/itkBSplineInterpolationWeightFunction.h:125:17: error: 'ZeroBasedIndexRange' was not declared in this scope

Thanks @dzenanz This error message must have been from a previous build. With force-push https://github.com/InsightSoftwareConsortium/ITK/compare/4a6a1de47f3da0563d6f39a34729cfe42a0792a1..ffc90b13d0f38b82c188dc2404611236ebf5b90f I added the missing `#include "itkIndexRange.h"`, which should fix the error. And indeed, on other platforms, this PR now builds fine!